### ISSUE TITLE
Ruffled acrosome

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#10312]
**Preferred term label:**
 PMA, many acrosomes had 

**Synonyms**

**Definition (free text, please give PubMed ID)**
An irregular wavy appearance of sperm acrosomes.
PMID: 12095502

The human acrosome and the acrosome reaction are important for human fertilization. Acrosome-intact sperm bind to the surface of the zona pellucida, where the physiological acrosome reaction is thought to be induced by zona pellucida glycoproteins. The proportion of sperm that react with acrosomes on the surface of the zona pellucida (the zona pellucida-induced acrosome reaction) is strongly correlated with the proportion of oocytes penetrated by sperm during in vitro fertilization.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

[OMIM:620499](https://omim.org/entry/620499):SPERMATOGENIC FAILURE 86; SPGF86
[OMIM:620500](https://omim.org/entry/620500): SPERMATOGENIC FAILURE 87; SPGF87

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000501
- label: Ruffled acrosome
- parents: [Abnormal sperm head morphology (HP:0012865)](https://hpo.jax.org/app/browse/term/HP:0012865)
- pmid: PMID:12095502
- def: An irregular wavy appearance of sperm acrosomes.
- comment: The human acrosome and the acrosome reaction are important for human fertilization. Acrosome-intact sperm bind to the surface of the zona pellucida, where the physiological acrosome reaction is thought to be induced by zona pellucida glycoproteins. The proportion of sperm that react with acrosomes on the surface of the zona pellucida (the zona pellucida-induced acrosome reaction) is strongly correlated with the proportion of oocytes penetrated by sperm during in vitro fertilization.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10312
- synonyms:
